planning/behavior_velocity_planner/src/utilization/interpolate.cpp

```diff
@@ -12,13 +12,98 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <interpolation/spline_interpolation.hpp>
 #include <utilization/interpolate.hpp>
 #include <utilization/util.hpp>
 
+#include <autoware_auto_planning_msgs/msg/path_point_with_lane_id.hpp>
+#include <autoware_auto_planning_msgs/msg/path_with_lane_id.hpp>
+
 #include <vector>
 
 namespace behavior_velocity_planner
 {
+bool splineInterpolate(
```

Comment:
In the original splineInterpolate function, the velocities are not interpolated.

I think that it is no problem because the velocities of interpolated path are not used in insersection/blind spot modules.
Is it right?

Reply:
@tkimura4 
>To my understanding, it is no problem because the velocities of interpolated path are not used in insersection/blind spot/no stopping area modules.
Is it right?

yes velocity is not used but I think it should be interpolated if it's interpolating path with lane id. because this function is not  interpolating pose array.